internal/api/front/perses/src/app/project/dashboard/dashboard.module.ts

```diff
@@ -12,16 +12,20 @@
 // limitations under the License.
 
 import { NgModule } from '@angular/core';
+import { DashboardRoutingModule } from './dashboard-routing.module';
 import { DashboardDetailsComponent } from './dashboard-details/dashboard-details.component';
+import { DashboardListComponent } from './dashboard-list/dashboard-list.component';
 import { SharedModule } from '../../shared/shared.module';
-import { DashboardRoutingModule } from './dashboard-routing.module';
-
+import { PageModule } from '../../shared/component/page/page.module';
+import { NgxChartsModule } from '@swimlane/ngx-charts';
 
 @NgModule({
-  declarations: [DashboardDetailsComponent],
+  declarations: [DashboardListComponent, DashboardDetailsComponent],
   imports: [
-    SharedModule,
+    NgxChartsModule,
     DashboardRoutingModule,
+    PageModule,
```

Comment:
I dont think you need to import the Page module here it should be in the shared module already no ?

Reply:
actually it is not. I had to do it on my side too (for the variable implementation). So I think we can keep it like that and I will remove it once I will integrate my changes with these ones.

Reply:
perhaps let's put a comment : 

```typescript
// TODO @Nexucis : to be removed once variable is integrated
```

Reply:
Nope it's needed as in prometheusrule.module, otherwise <app-page> & similar stuffs dont work

Or there's a way to propagate the import somehow ?

Reply:
yeah because it was in pending during my review. It's weird that my comments come after yours....

Reply:
TODO has been added, I'm resolving the conversation